homeassistant/helpers/state.py

```diff
@@ -53,36 +70,67 @@ def reproduce_state(hass, states, blocking=False):
 
     to_call = defaultdict(list)
 
+    # Update this dict of lists when new services are added to HA.
+    # Each item is a service with a list of required attributes.
+    service_attributes = {
+        SERVICE_PLAY_MEDIA: [ATTR_MEDIA_CONTENT_TYPE, ATTR_MEDIA_CONTENT_ID],
+        SERVICE_MEDIA_SEEK: [ATTR_MEDIA_SEEK_POSITION],
+        SERVICE_VOLUME_MUTE: [ATTR_MEDIA_VOLUME_MUTED],
+        SERVICE_VOLUME_SET: [ATTR_MEDIA_VOLUME_LEVEL],
+        SERVICE_NOTIFY: [ATTR_MESSAGE],
+        SERVICE_SET_AWAY_MODE: [ATTR_AWAY_MODE],
+        SERVICE_SET_FAN_MODE: [ATTR_FAN],
+        SERVICE_SET_TEMPERATURE: [ATTR_TEMPERATURE],
+    }
+
+    # Update this dict when new services are added to HA.
+    # Each item is a service with a corresponding state.
+    service_to_state = {
```

Comment:
Should I move the dicts out to the module level and make them global constants instead?